pkg/transformer/kubernetes/kubernetes.go

```diff
@@ -418,6 +418,35 @@ func (k *Kubernetes) CreateKubernetesObjects(name string, service kobject.Servic
 	return objects
 }
 
+func BuildImage(name string, service kobject.ServiceConfig, composeFileDir string) string {
```

Comment:
Same as with Execute in transformer. I think that it would be great to have all the image building and handling in some nice separate package.